From c67e4b130f00ab417b8baaaae6741f459b84a4c4 Mon Sep 17 00:00:00 2001
From: tonishi92 <tatsuo.onishi@gmail.com>
Date: Tue, 13 Dec 2022 15:21:35 +0100
Subject: [PATCH 48/52] (Patch) Fix potential bug in mosaic_settling aer phase

Description:
In module_mosaic_settling, set the aerosol interstitial phase to 1
explicitly instead of using the ai_phase variable, in case ai_phase is
not set properly when the settling module is first called.

Source: L. Marelle

List of modified files:
 chem/module_mosaic_settling.F
---
 chem/module_mosaic_settling.F | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/chem/module_mosaic_settling.F b/chem/module_mosaic_settling.F
index b03016a4..fe371110 100755
--- a/chem/module_mosaic_settling.F
+++ b/chem/module_mosaic_settling.F
@@ -52,7 +52,7 @@
     USE module_data_mosaic_asect, ONLY:  &
         dens_water_aer, mw_water_aer, dens_aer, & 
         dlo_sect, dcen_sect, dhi_sect, volumlo_sect, &
-        volumhi_sect, ai_phase, ncomp_aer, nsize_aer, ntype_aer, &
+        volumhi_sect, ncomp_aer, nsize_aer, ntype_aer, &
         numptr_aer, waterptr_aer, massptr_aer, lptr_so4_aer
     USE module_configure, ONLY:  num_chem
     USE module_data_mosaic_other, ONLY: pi 
@@ -153,8 +153,8 @@
 
 !   Get aerosol wet density and wet diameter
 
-!   *** for now, just calc vdep_aer for iphase = ai_phase !lm, check in mixactivate how cw aerosols are removed
-    iphase = ai_phase
+!   *** for now, just calc vdep_aer for iphase = 1 
+    iphase = 1 
 
     do itype = 1, ntype_aer
     do n = 1, nsize_aer(itype)
-- 
2.31.1

